src/_styles/shared/layout.less

```diff
@@ -68,16 +68,17 @@ main {
 }
 
 .parallax {
-  position: absolute; // added for pseudo-element
-  height: 100%;
-  width: 100%;
+  position: fixed; // added for pseudo-element
+  height: 300%;
+  width: 200%;
   top: 0;
   left: 0;
   z-index: -1;
   clip: rect(auto, auto, auto, auto);
 
   // Fixed-position background image
   > .parallax-image {
+    background-image: url(denver-kitchen-remodel.jpg);
```

Comment:
If another element were to have the class `.parallax-image` it'll also have to get this `background-image`. We should only apply this specific `background-image` to an id target of some sort, possibly `#top-splash`